includes/database/tables/class-order-adjustments.php

```diff
@@ -145,4 +147,29 @@ protected function __202002141() {
 
 	}
 
+	/**
+	 * Upgrade to version 202011120
+	 *  - Change default value to `NULL` for `type_id` column.
+	 *  - Add `type_key` column.
+	 *
+	 * @since 3.0
+	 * @return bool
+	 */
+	protected function __202011120() {
+
+		// Update `type_id`.
+		$result = $this->get_db()->query( "
+			ALTER TABLE {$this->table_name} MODIFY COLUMN `type_id` bigint(20) default NULL;
+		" );
+
+		// Add `type_key`.
+		$column_exists = $this->column_exists( 'type_key' );
+		if ( false === $column_exists ) {
+			$result = $this->get_db()->query( "
+				ALTER TABLE {$this->table_name} ADD COLUMN `type_key` varchar(32) default '' AFTER `type_id`;
+			" );
+		}
```

Comment:
```suggestion
		}
		
		// Change `type_id` with `0` value to `null` to support new default.
		$this->get_db()->query( "UPDATE {$this->table_name} SET type_id = null WHERE type_id = 0;" )
```

To convert old `0` values to `null`.